lib/matplotlib/animation.py

```diff
@@ -1271,28 +1459,44 @@ def to_html5_video(self):
             # Now open and base64 encode
             with open(f.name, 'rb') as video:
                 vid64 = encodebytes(video.read())
-                self._base64_video = vid64.decode('ascii')
-                self._video_size = 'width="{0}" height="{1}"'.format(
-                        *writer.frame_size)
+                vid_len = len(vid64)
+                if vid_len >= embed_limit:
+                    import warnings
+                    warnings.warn('Animation movie is {0} bytes, exceeding '
+                            'the limit of {1}. If you\'re sure you want a '
```

Comment:
Use double quotes around and you won't need to escape single quotes.
No need to number the "{}"'s.

Reply:
Done.